### ISSUE TITLE
Filters not displayed in UI - Serivce Manager invokableClasses property issue

### ISSUE THREAD

**Issue by sandrokeil** [zfcampus/zf-apigility-admin#337]
The class `\ZF\Apigility\Admin\Model\AbstractPluginManagerModel` uses the [reflection property](https://github.com/zfcampus/zf-apigility-admin/blob/master/src/Model/AbstractPluginManagerModel.php#L55 "Source Code") `invokableClasses` of the plugin manager class, in this case `\Zend\Filter\FilterPluginManager`. But there is no property in `Zend\Filter` version 2.6.1. Validators works because of map definitions.

A better solution would be to use the `\Zend\ServiceManager\ServiceManager::getRegisteredServices()` method and use the class values to remove duplicated definitions.

@weierophinney Not sure if this is a slightly BC break due the v3 components compatibility.

**Comment by metanav** [comment]
The issue is still there. Any progress?

**Comment by kelsey9649** [comment]
I did a quick rollback on a fork to use the `Config`'s `filter_metadata`. While it will not dynamically update with registered filters in the system it has a listing of all the Zend filters. While debugging I did not see any filters registered with the `FilterPluginManager`.  Fork is [here](https://github.com/Truman-State-University/zf-apigility-admin/commit/548c5da914cdb4f3cd10f23a46bcf2ba74447f99)
